ap/calibration/seeds.py

```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
+
+"""
+Methods related to creating event and object seeds during calibration.
+"""
+
+import hashlib
+
+from ap.calibration import calibrator
+from ap.util import maybe_import, primes
+from ap.columnar_util import Route, get_ak_indices, set_ak_column, has_ak_column
+
+np = maybe_import("numpy")
+ak = maybe_import("awkward")
+
+
+@calibrator(
+    uses={
+        # global columns for event seed
+        "event", "nGenJet", "nGenPart", "nJet", "nPhoton", "nMuon", "nElectron", "nTau", "nSV",
+        # first-object columns for event seed
+        "Tau.jetIdx", "Tau.decayMode",
+        "Muon.jetIdx", "Muon.nStations",
+        "Jet.nConstituents", "Jet.nElectrons", "Jet.nMuons",
```

Comment:
Wouldn't it be better/more robust/more consistent to completely rely on Routes here?